theme.json

```diff
@@ -95,7 +96,13 @@
 	},
 	"styles": {
 		"spacing": {
-			"blockGap": "1.5em"
+			"blockGap": "calc( var( --wp--style--block-gap, 1.5em ) * 2 )",
```

Comment:
Is this a very large gap? Seems like it would come out to 3em. Are any of the child themes still configured to the old `0.75em` gap?